src/data/reducers/assets.js

```diff
@@ -0,0 +1,47 @@
+import { REQUEST_ASSETS_SUCCESS, DELETE_ASSET_SUCCESS, ASSET_XHR_FAILURE, FILTER_UPDATED } from '../actions/assets';
```

Comment:
Is there a better way to do this? This is not gonna be very pretty as we have more actions.

Reply:
You can do something like `import * as AssetsActions from '../actions/assets';` -- then when referencing below do something like `AssetsActions.DELETE_ASSET_SUCCESS`

Reply:
I think @fysheets was going to propose moving the action objects out of actions/assets.js into something like actions/assetsActionTypes.js and actions creators into assetsActionCreators.js. If we move in that direction, we could do import * without importing the action creators too. Would it make sense to do that, assuming that recommendation is how we want to move forward? I don't think it would be a big pain to switch between the two if they're being used the same way in the assets reducer for the time being.